src/components/Clock.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+
+type State = {
+  time: string;
+};
+
+type Props = {
+  name: string;
+};
+
+export class Clock extends React.Component<Props, State> {
+  state: State = {
+    time: '',
```

Comment:
```suggestion
    time: new Date().toLocaleTimeString(),
```
Let's give initial value because now we have no time displayed when your app is loaded

Reply:
I've actually did this at the beginning but was i bit confused by task description 😆